### PR TITLE
fix: change swap-line shortcuts on macos

### LINKED ISSUE #863: Swap-line shortcuts don't work on macOS
**Describe the bug**

> When I try to swap lines with the command that CP Editor has configured in macOS (Command+Shift+(Up or Down Arrow)) it doesn't work, because in macOS, the shortcut Command+Up Arrow for example when used in text, moves the cursor from the current cursor position to the beginning, and Command+Down to the end
>
> So when you also use Shift, what you get is that you select all the way to the beginning or all the way to the end, depending on the direction you have chosen.
>
> The thing is that this shortcut is native to macOS and takes precedence over CP Editor, i.e. it never swaps lines.

**Additional context**

> what is the possibility to change the shortcut by another key combination like in Sublime Text(Command+Cntrl+Up arrow) or VSCode(Alt+Up arrow)

#862

### PR DESCRIPTION
**Pull request by ouuan**
## Description

Change swap-line shortcuts as requested in #863.

## Related Issues / Pull Requests

This fixes #863.

## How Has This Been Tested?

@swift-zym Can you test this?

## Checklist

- [x] If the key of a setting is changed, the `old` attribute is updated or it is resolved in SettingsUpdater.
- [x] If there are changes of the text displayed in the UI, they are wrapped in `tr()` or `QCoreApplication::translate()`.
- [x] If needed, I have opened a pull request or an issue to update the [documentation](https://github.com/cpeditor/cpeditor.github.io).
- [x] If these changes are notable, they are documented in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by swift-zym** [comment]
@ouuan It seems didn't work well on my Mac.

File: src/appwindow.cpp [hunk 1]
```diff
@@ -62,6 +62,11 @@ AppWindow::AppWindow(bool noHotExit, QWidget *parent) : QMainWindow(parent), ui(
     allocate();
     setConnections();
 
+#ifdef Q_OS_MACOS
+    ui->actionSwapLineUp->setShortcut({"Command+Ctrl+Up"});
+    ui->actionSwapLineDown->setShortcut({"Command+Ctrl+Down"});
```
Comment (swift-zym) [inline src/appwindow.cpp:67]: Maybe you should change "Ctrl" to "Control".

**Review by swift-zym — COMMENTED** [review]
I don't know why... but it still didn't work.

**Review by swift-zym — APPROVED** [review]
Works well now.

**Review by coder3101 — APPROVED** [review]
Command+Control+Up/Down works fine on MBP macOS Catalina

**Review by swift-zym — APPROVED** [review]

**Review by coder3101 — APPROVED** [review]

**Comment by coder3101** [comment]
@ouuan `v6.9` is 4 commits ahead `master`, please create a PR for master from that version branch.

**Comment by ouuan** [comment]
> @ouuan `v6.9` is 4 commits ahead `master`, please create a PR for master from that version branch.

6.9.2 will come soon.

**Comment by ouuan** [comment]
> 6.9.2 will come soon.

Shall we wait for snap release?

**Comment by coder3101** [comment]
> > 6.9.2 will come soon.
> 
> Shall we wait for #835?

Sure

**Comment by swift-zym** [comment]
Why #863 still open?

**Comment by coder3101** [comment]
> Why #863 still open?

Because not merged to master.

**Pull request merged**